src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -265,4 +267,50 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
 
         return state;
     }
+
+    private async _processAdditionalAction(error: unknown, shouldSetAsDefaultInstrument: boolean, instrumentId: string, methodId: string): Promise<InternalCheckoutSelectors> {
+        if (!(error instanceof RequestError) || !some(error.body.errors, {code: 'additional_action_required'})) {
```

Comment:
Do we have to figure out _which action_ is required? this seems like it could cause issues in the future if we try to add new actions 🤔

Reply:
Yes I'm agree, and I just followed the same pattern as: 

- Adyen V2: https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts#L354
- Stripe v3: https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/payment/strategies/stripev3/stripev3-payment-strategy.ts#L151
- External: https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/payment/strategies/external/external-payment-strategy.ts#L59

and there are more payment providers that follows this pattern, but I'm totally agree, we need to improve this type of patterns in a better solution.

Reply:
Can you think of a way of improving this? It'll be good if we could avoid these situations going forward.

Reply:
Yes, I'll talk with Kyle to create a ticket to attend this specific request, I'll let you know once the ticket is created

Reply:
I was hoping we could solve this instance of the issue within this PR. There's no reason to get it merge this way...